### PR TITLE
create contact js

### PR DESCRIPTION
**Pull request by HebaAhmad1**
relates #32

### CODE REVIEW

**Review by AbdallahGot — DISMISSED** [review]

**Review by NadaSaleh20 — DISMISSED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

**Review by Rawanzaqout — DISMISSED** [review]

File: contact/contact.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+        var icon = document.getElementById("icon");
```
Comment (Ibrahim-Jarada) [inline contact/contact.js:2]: Try to avoid using var as you can, its preferable to use const here instead of var

File: contact/contact.css [hunk 2]
```diff
@@ -88,4 +85,70 @@ header .dark-mode{
     margin-top: 40px;
     color: #ff9500;
     text-decoration: none;
+}
+
+@media only screen and (min-width :1300px){
```
Comment (AbdallahGot) [inline contact/contact.css:90]: image
!!!!!!!!!!!
why it looks like that !! try to improve it

**Review by AbdallahGot — CHANGES_REQUESTED** [review]

File: contact/contact.css (hunk 2 above)
Comment (HebaAhmad1) [reply contact/contact.css:90]: Done

File: contact/contact.js (hunk 1 above)
Comment (HebaAhmad1) [reply contact/contact.js]: Done

**Review by Ibrahim-Jarada — APPROVED** [review]

**Review by AbdallahGot — APPROVED** [review]

**Review by MontherIsmail — APPROVED** [review]

**Review by Rawanzaqout — APPROVED** [review]

**Review by NadaSaleh20 — APPROVED** [review]

**Pull request merged**